odl/util/graphics.py

```diff
@@ -428,7 +428,7 @@ def show_discrete_data(values, grid, title=None, method='',
                 # If it exists and we should update it
                 csub.colorbar.set_clim(minval, maxval)
                 csub.colorbar.set_ticks(ticks)
-                csub.colorbar.set_ticklabels([fmt.format(*ticks)])
+                csub.colorbar.set_ticklabels([fmt % t for t in ticks])
```

Comment:
Why would we want to use old style formatting here?

Reply:
The `_colorbar_format` helper returns an old-style format string because `plt.colorbar(..., format=fmt)` expects it. So either we make a separate helper or just also use the old-style format here.

Reply:
damnit, I guess we'll have to live with it then

Reply:
Sad but true.

Is that "Approve"?

Reply:
Sure!